src/main/java/com/autotune/analyzer/kruizeObject/CreateExperimentConfigBean.java

```diff
@@ -0,0 +1,93 @@
+package com.autotune.analyzer.kruizeObject;
+
+public class CreateExperimentConfigBean {
```

Comment:
Why is this needed? Dont we already have Experiment related classes? Please add a javadoc with sufficient explanation

Reply:
This object is placeholder for create Experiment template which holds default values which is set in 

https://github.com/kruize/autotune/blob/ffe14a9eec97a359bc877cd8759021f763b9ead1/src/main/java/com/autotune/utils/KruizeConstants.java#L780C1-L788C70

Reply:
Add javadoc with explination